ext/third-party/sparsepp/README.md

```diff
@@ -1,17 +1,21 @@
-[![Build Status](https://travis-ci.org/greg7mdp/sparsepp.svg?branch=master)](https://travis-ci.org/greg7mdp/sparsepp)
```

Comment:
This whole third-party library has been updated to latest source code from https://github.com/greg7mdp/sparsepp